server/routes/overview/overviewController.ts

```diff
@@ -99,6 +101,31 @@ export default class OverviewController {
     const { prisonerSummary } = req.session
 
     const supportNeeds = await this.curiousService.getPrisonerSupportNeeds(prisonNumber, req.user.username)
+
+    // Loop through the healthAndSupport needs array and update the prison name for each need
+    await Promise.all(
+      supportNeeds.healthAndSupportNeeds.map(async supportNeed => {
+        const prison = await this.prisonService.getPrisonByPrisonId(supportNeed.prisonId, req.user.token)
+        if (prison) {
```

Comment:
I may not have written this prisonService and timelineService in the best way originally 😬 , but by calling the prisonService directly like this you are missing a trick in respect of try/catch error handling.
As you have it at the moment if there is an error calling the prisonService the error is not caught and the page will fail (and it might even crash the node app! 😬 )
Have a look at how the timeline controller does this, but there is an intermediate method that it calls (in timelineService) called `lookupPrison` which does the try/catch

I'm not suggesting this method for getting support needs needs to use the timelineService (that would be wrong), but have a look at how it currently works and see if you can do something similar here
(hint - it might be a case of moving `lookupPrison` out of timelineService and into prisonService so that it can be used by things other than the timeline)